imports/plugins/core/checkout/server/i18n/fr.json

```diff
@@ -10,13 +10,13 @@
         "yourCart": "Votre panier",
         "discountTotal": "Remise totale",
         "orderTotal": "Total de la commande",
-        "fulfilledBy": "Réalisé par",
+        "fulfilledBy": "Effectué par",
         "quantityTotal": "Quantité totale",
         "orderSubtotal": "Total partiel",
-        "orderShipping": "livraison",
+        "orderShipping": "Livraison",
         "orderTax": "Paramètres de taxe",
-        "trackYourDelivery": "Les mises à jour concernant la commande seront envoyées à",
-        "registerGuest": "salut! Ajouter un email et recevoir des mises à jour de commande"
+        "trackYourDelivery": "Les informations concernant la commande seront envoyées à",
+        "registerGuest": "Bonjour ! Renseignez votre adresse e-mail pour être informé de l'avancement de votre commande"
```

Comment:
Honest question here -- is it normal in French to put spacing before punctuation like `!`, `?`, etc.?  I see you've done this a fair bit but similar exists in places in the original translations.

Reply:
That's one of the weirdest little-know quirks of the French language! We do put a space before what I'd call "double punctuation" (that is, any punctuation composed of two parts, such as "!", "?", ":" and ";"). However, we don't put spaces before periods or commas.

I'd have a hard time explaining why. I've seen Canadians often do the same though.

Reply:
I'll be honest, it does read slightly better and I do it subconsciously from time to time (Canadian here).  Thanks for the info!  :)